backend/openshift.deploy.yml

```diff
@@ -41,7 +41,31 @@ parameters:
   - name: PROMOTE
     description: Image (namespace/name:tag) to promote/import
     value: bcgov/nr-fam-idim-lookup-proxy/backend:test
+  - name: IDIM_WEB_SERVICE_URL
+    description: The url of the IDIM web serivce
+    required: true
+  - name: IDIM_WEB_SERVICE_ID
+    description: The service id of the IDIM web serivce
+    required: true
+  - name: IDIM_WEB_SERVICE_USERNAME
+    description: The username of the IDIM web serivce
+    required: true
+  - name: IDIM_WEB_SERVICE_PASSWORD
+    description: The password of the IDIM web serivce
+    required: true
+  - name: API_KEY
+    description: The bearer token in order to call the api
```

Comment:
Description needs to be adjusted.

Reply:
thanks so much Ian!! I forgot this :  )))